### ISSUE TITLE
Wrong bot toml path has bad error message

### ISSUE THREAD

**Issue by NicEastvillage** [RLBot/core#56]
> If you mess up your path to your bot in your toml, at least in Python, you get a very quiet "the path does not exist" or whatever and that's it. No hint about what path or what the issue is
> I can't even tell what bot it's coming from, what file, what line of what code, I get zero info. It's gonna be an issue
> \- kaiyotech

**Comment by swz-git** [comment]
The error message comes from `cmd.exe` and we can't really change it. What we should do imo is log a big red error message when a bot exits with a non-zero exit code. That way you can pretty easily figure out why it crashed by looking at the line above.

**Comment by Kaiyotech** [comment]
I mean, you could check path before attempting to parse it, so you could say "failed to find path xxx" if it's going to fail. Or you could just print "Parsing toml" and then when it fails that at least points to the toml file for the bot that failed (if we add your red warning), but I definitely appreciate that the console is relatively quiet currently and we don't want a bunch of "checking. Complete, checking, complete" garbage.

**Comment by swz-git** [comment]
That doesn't work because `run_command` doesn't need to be a path. Its a batch command (or /bin/sh command on linux). Though we could still point to the toml path, `agent_id` and/or the `run_command` of the bot that failed when it fails in the error message I suggested above.

**Comment by Kaiyotech** [comment]
Ok. I see. Because the run command can be complex we can't really check it beforehand to make sure it's valid. We could, but it's a mess probably. Just something that at least points to the toml would be enough.

**Comment by VirxEC** [comment]
I agree with swz, the best we can probably do is a big bold red error message "{bot/script name}'s process exited with a non-0 status" to make it a bit more clear

**Comment by VirxEC** [comment]
![Image](https://github.com/user-attachments/assets/8cd204c1-c008-4daf-ac3a-f7ee0314fb8f)
